source/includes/_addax.md

```diff
@@ -0,0 +1,186 @@
+```
+
+You can view past transactions using a request like:
+
+```shell
+curl "https://api.zinc.io/v1/addax/transactions?count=100&offset=0" -u <client_token>:
+```
+
+## Create an order
+
+Follow the [standard documentation for creating orders]() with two changes:
+
+1. Add `"addax": true` to your request body.
+2. Do not supply the `retailer_credentials`, `payment_method`, or `billing_address` keys.
+
+All other ordering features and options work as specified in the standard
+documentation.
+
+Successful orders will result in a transaction decreasing your account balance
+for the currency of the market. Failed orders will not cost you anything.
+
+## Cancelling orders
+
+The Zinc API supports pre-shipment order cancellation on Amazon.com and
+Amazon.co.uk. Simply POST to the cancellation endpoint:
+
+```shell
+curl "https://api.zinc.io/v1/orders/<order_id>/cancel" \
+  -X POST \
+  -u <client_token>: \
+  -H 'Content-type: application/json' \
+  -d '{
+    "webhooks": {
+      "request_succeeded": "https://www.example.com/webhooks/success",
+      "request_failed": "https://www.example.com/webhooks/failed"
+     }
+   }'
+```
+
+The `request_succeeded` and `request_failed` webhooks are optional. If supplied,
+they will be called when the corresponding event occurs on the cancellation
+request.
+
+### `attempting_to_cancel`
+
+In about 50% of cases, Amazon is unable to immediately cancel an order. Instead,
+they tell Zinc that they're "Attempting to Cancel" the order. This currently
+results in a _failure_ code in the API. We hope to improve this in the future to
+automatically update when the order is either cancelled or shipped. At the
+moment, however, you'll need to manually open a refund request for these cases.
+See below for more details.
+
+## Returning orders
+
+The Zinc API also supports generating return labels through Amazon. Here's an
+example request:
+
+```shell
+curl "https://api.zinc.io/v1/orders/<order_id>/return" \
+  -X POST \
+  -u <client_token>: \
+  -H 'Content-type: application/json' \
+  -d '{
+    "webhooks": {
+      "request_succeeded": "https://www.exaple.com/webhooks/return/success",
+      "request_failed": "https://www.exaple.com/webhooks/return/failed",
+    },
+    "products": [{"product_id": "B0000001234", "quantity": 1}],
+    "reason_code": "inaccurate website description",
+    "method_code": "UPS Dropoff",
+    "explanation": "Additional details for Amazon seller",
+  }'
+```
+
+Webhooks are optional, as usual, but the other four fields are required.
+
+`products` is in the same format as for an order. It determines which products
+of a multi-item order to return.
+
+`reason_code` and `method_code` are passed directly to Amazon. The available
+options may vary based on Amazon country and customer address. However, `UPS
+Dropoff` is the only method supported for automatic refunds. If you'd like to
+use a different method, you'll need to manually request refunds as specified
+below.
```

Comment:
Should we put something in here to say that people should default to using "inaccurate website description" for US and whatever we use for UK?